ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -123,22 +136,27 @@ public TaskTracker tasks() throws SemanticException {
      behave like a noop or a pure MD alter.
   */
       if (table == null) {
-        return newTableTasks(tableDesc);
+        newTableTasks(tableDesc);
       } else {
-        return existingTableTasks(tableDesc, table, replicationSpec);
+        existingTableTasks(tableDesc, table, replicationSpec);
+      }
+
+      if (!isPartitioned(tableDesc)) {
+        createTableReplLogTask(tableDesc.getTableName(), tableDesc.tableType());
```

Comment:
tracker is not taking into account this new task being added.

Reply:
I think, it is ok, because this is the last task being added in the current table's DAG. When the caller adds this to loadTaskTracker during post-processing, it will be accounted.